server/token_test.go

```diff
@@ -1585,3 +1585,28 @@ func TestServeTokenWithClientValidation(t *testing.T) {
 		})
 	}
 }
+
+func TestTokenCORSHeaders(t *testing.T) {
+	s := &IDPServer{
+		serverURL: "https://idp.test.ts.net",
+	}
+	req := httptest.NewRequest("OPTIONS", "/token", nil)
+	rr := httptest.NewRecorder()
+	s.ServeHTTP(rr, req)
+
+	ao := rr.Header().Get("Access-Control-Allow-Origin")
+	am := rr.Header().Get("Access-Control-Allow-Method")
```

Comment:
just noticed: it's `Access-Control-Allow-Methods`, not `Access-Control-Allow-Method` https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/CORS#access-control-allow-headers